### PR TITLE
[INV-103] Create My Bookings page

### PR DESCRIPTION
**Pull request by Savesh1508**
# Pull request for: [INV-103]

## Description

- Added My Bookings page
- Added Categories for bookings
- Added shared helper for capitalize letters

https://github.com/user-attachments/assets/ee9962c3-a2c5-43a2-8e5f-1fc8364ecbd2

## Checklist

- [x] I have manually tested the changes and related features
- [x] I have run the linter and fixed any possible issues
- [x] I have run the build and fixed any possible issues

### CODE REVIEW

File: src/modules/bookings/components/bookings-navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+interface BookingsNavbarProps {
+  selectedCategory: BookingStatus;
+  onCategoryChange: (status: BookingStatus) => void;
+}
+
+const styles = {
+  navItem: (isSelected: boolean) => ({
+    width: '100%',
+    textAlign: 'center',
+    borderBottom: isSelected ? '3px solid #000' : '1px solid #e9e9e9',
```
Comment (SurajSulten) [inline src/modules/bookings/components/bookings-navbar.tsx]: Maybe it is better to make border bottom pink

File: src/modules/bookings/constants/booking-status.constant.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export enum BookingStatus {
+  Pending = 'pending',
+  Active = 'active',
+  Inactive = 'inactive',
+  Completed = 'completed',
+  Upcoming = 'upcoming',
+  Cancelled = 'cancelled',
```
Comment (ShahobiddinIsmoilov) [inline src/modules/bookings/constants/booking-status.constant.ts]: Please use ```canceled``` as backend

File: src/shared/helper/capitalizer.helper.ts
Comment (ShahobiddinIsmoilov) [inline src/shared/helper/capitalizer.helper.ts:1]: Please use util postfix and move the file in utils folder

File: src/modules/bookings/components/bookings-navbar.tsx (hunk 1 above)
Comment (ShahobiddinIsmoilov) [reply src/modules/bookings/components/bookings-navbar.tsx]: > Maybe it is better to make border bottom pink

Agreed. Style consistency

**Review by ShahobiddinIsmoilov — CHANGES_REQUESTED** [review]

File: src/modules/bookings/components/bookings-navbar.tsx (hunk 1 above)
Comment (Savesh1508) [reply src/modules/bookings/components/bookings-navbar.tsx]: Fixed

File: src/shared/helper/capitalizer.helper.ts
Comment (Savesh1508) [reply src/shared/helper/capitalizer.helper.ts:1]: Fixed

File: src/modules/bookings/constants/booking-status.constant.ts (hunk 2 above)
Comment (Savesh1508) [reply src/modules/bookings/constants/booking-status.constant.ts]: Fixed

**Review by ShahobiddinIsmoilov — APPROVED** [review]

**Review by farrooh — APPROVED** [review]

**Pull request merged**